sdk/src/client/api/block_builder/transaction.rs

```diff
@@ -12,11 +12,12 @@ use crate::{
         payload::transaction::{RegularTransactionEssence, TransactionPayload},
         semantic::{semantic_validation, TransactionFailureReason, ValidationContext},
         signature::Ed25519Signature,
-        Block, BlockId,
+        BlockId, BlockWrapper,
     },
 };
 
-const MAX_TX_LENGTH_FOR_BLOCK_WITH_8_PARENTS: usize = Block::LENGTH_MAX - Block::LENGTH_MIN - (7 * BlockId::LENGTH);
+const MAX_TX_LENGTH_FOR_BLOCK_WITH_8_PARENTS: usize =
```

Comment:
Put a TODO, with the three different parent types this can be wrong now

Reply:
Added